_posts/Illumination_Correction_A_Comparison_of_Methods.md

```diff
@@ -0,0 +1,135 @@
+
+These methods are all different in their approach and how they are accessed/used. 
+All of these use a “retrospective” approach, which can derive an illumination correction function using the images directly.
+In contrast, a “prospective” approach requires that during the image acquisition stage that a dark image (background without light) and bright image (background with light) must be taken at each site. 
+These images then can be used to derive an illumination correction function.
+This means that any package or software that uses this approach would be applied at the beginning of the experiment which isn’t always feasible.
+This is what makes the retrospective approach much better since you can use it on publically-available data and do not need to make the image acquisition step longer (it’s more convenient).
+
+In this post, I created the pros and cons based on what I have researched and then my own personal experiences using these methods as seen in Figure 1.
+
+![Pros and Cons](analysis/Pros_and_Cons_Table.png)
+
+*Figure 1. Table of Illumination Correction Methods*
+
+**Let's go through each of these methods one by one:**
+
+I started off my research with CellProfiler (CP). 
+Whenever I searched for illumination correction methods, this software came up first. 
+The main way that it is used is through the GUI (graphical user interface).
+There are ways to utilize CP through Python and Jupyter notebooks (see [<u>https://github.com/CellProfiler/notebooks</u>](https://github.com/CellProfiler/notebooks)), but the most recent examples are from 2-5 years ago.
+
+***Side-note:*** If you aren’t familiar with the coding world, it is important to know that software changes frequently and can become out of date even after less than a year. 
+Please use that resource I provided with a grain of salt as it very well could be out of date or work perfectly!
+
+Back to the GUI, CP 4.0 has a very user-friendly interface that makes it easy for non-data science biologists to use as well as data science beginners. 
+But this comes with a not-so-fun challenge that impacts generalizability, which is manual parameters (I know, terrifying!).
+Now, a few manual parameters are fine within a software, like if you need to make minor corrections to fit the function to your data, but too many manual parameters creates many avenues for confusion and error. 
+When it comes to CP illumination correction, it has MANY manual parameters (ranging from 5 to 15 depending on the method you choose).
+As a beginner in the field, how are you meant to know which threshold to use or if the correction should be run based on all images or per image? 
+That is my main issue with CP. It is set up to be easy to use for beginners, but you need to be an expert to use it properly.
+
+There is [documentation](https://cellprofiler-manual.s3.amazonaws.com/CellProfiler-4.2.1/index.html) on the modules and parameters that can help with trying to understand the IC function you are creating. 
```

Comment:
I like the abbreviation `IC`, but I didn't catch where you defined it first.

I'd recommend sticking with either 1) writing it out fully each time, or 2) to define the acronym the first time you use the phrase `illumination correction` and then use IC every time.

I lean toward option 1!

Reply:
@gwaybio I do define the abbreviation in the first sentence under the "What is illumination correction and why is it used?" section of the blog. 😊